File: j2k/testData/fileOrElement/postProcessing/IfToElvis.java
class C {
    String foo1(String s) {
        return s != null ? s : "";
    }

    String foo2(String s) {
        return s == null ? s : "";
    }

    String foo3(String s) {
        return s != null ? "" : s;
    }

    String foo4(String s) {
        return s == null ? "" : s;
    }

    String foo5(String s) {
        return null == s ? "" : s;
    }

    String foo6(String s) {
        return null == s ? s : s;
    }
}

File: j2k/testData/fileOrElement/postProcessing/IfToElvis.kt
internal class C {
    fun foo1(s: String?): String {
        return s ?: ""
    }

    fun foo2(s: String?): String? {
        return if (s == null) s else ""
    }

    fun foo3(s: String?): String? {
        return if (s != null) "" else s
    }

    fun foo4(s: String?): String {
        return s ?: ""
    }

    fun foo5(s: String?): String {
        return s ?: ""
    }

    fun foo6(s: String?): String? {
        return s ?: s
    }
}